.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,36 @@
+name: "Updatecli: Dependency Management"
+
+on:
+  schedule:
+    # Run every hour
+    - cron: '0 * * * *'
```

Comment:
I think this is a bit too aggressive, we might want to dial this back considering we will need to validate the compatibility of dependencies and wait for CI to finish. I think every day, similar to the proposed dependabot schedule is better to start with.

Reply:
Sure, we can change that as we wish. Running at 12 PM would be good? Or do you see a better time?

Reply:
I think 6pm GMT (just after daily) is best for our team, I don't know what timezone that translates to for this though.

Reply:
IIRC it's UTC/GMT, so 06 PM GMT is fine.